test/libsolidity/syntaxTests/parsing/while_loop.sol

```diff
@@ -1,9 +1,10 @@
 contract test {
     function fun(uint256 a) {
-        while (true) { uint256 x = 1; break; continue; } x = 9;
+        uint256 x;
+        while (true) { x = 1; break; continue; } x = 9;
     }
 }
 // ----
-// Warning: (20-115): No visibility specified. Defaulting to "public". 
+// Warning: (20-126): No visibility specified. Defaulting to "public". 
 // Warning: (33-42): Unused function parameter. Remove or comment out the variable name to silence this warning.
```

Comment:
Why does ``fun`` have an argument for this test anyways? Doesn't seem related to or necessary for the test...